keps/sig-cli/kustomize-secret-generator-plugins.md

```diff
@@ -179,12 +228,24 @@ func Env(root string, args []string) ([]kv.Pair, error) {
 }
 ```
 
-Users would install the plugin with `go build -buildmode=plugin ~/.config/kustomization_plugins/myplugin.so`
+Users would install the goplugin with:
```

Comment:
This should really only be for distributions created by organizations.  We should say a distribution could be created this way.  Publishing containers that have kustomize in them may be a good way of accomplishing this.

Reply:
added